tests/serialize_test.go

```diff
@@ -81,20 +74,20 @@ func Test_serialize(t *testing.T) {
 			},
 			want: map[string]any{
 				"r": map[string]any{
-					"newest": map[string]any{
-						"mode":     "drwxr-xr-x",
-						"modified": testDateTime,
-						"name":     "test",
-						"size":     10,
-					},
-					"files": []map[string]any{
-						{
+					"files": []any{
+						map[string]any{
 							"name":     "test",
-							"size":     10,
+							"size":     int64(10),
 							"mode":     "drwxr-xr-x",
-							"modified": testDateTime,
+							"modified": testDate,
```

Comment:
@moshloop do we want this to be `time.Time`? I don't think any nested `time.Time` or `time.Duration` are preserved.

Reply:
They are preserved for cel here: https://github.com/flanksource/gomplate/blob/main/serialize.go#L34

Reply:
They aren't preserved for struct fields like in this test case.